### DISCUSSION TITLE
如何给数据库改名以便在grafana中分开查看

### DISCUSSION THREAD

**Discussion by imtipi** [juicedata/juicefs#4338] [Q&A] [answered]
因为官方提供的grafana dashboard只有name进行筛选，请问如何修改数据库名字以便解决以前数据库重名冲突的问题，或者可以提供一些其他分开在grafana dashboard查看数据的建议吗

**Comment by zhijian-pro** [comment]
能否详细举例说明下 或者截图下

**Reply by imtipi** [reply]
![image](https://github.com/juicedata/juicefs/assets/27230181/ebece5d7-154f-4feb-8547-6b460707aa38)
这个面板来自 https://juicefs.com/docs/zh/community/administration/monitoring/ 的 [grafana_template.json](https://github.com/juicedata/juicefs/blob/main/docs/en/grafana_template.json)

这个name实际为vol_name（Volume 名称） https://juicefs.com/docs/zh/community/reference/p8s_metrics/

在最开始创建时我不知道使用面板进行监控，现在多台机器都是使用SQLite进行单机部署，同时数据库文件名也一模一样，导致在后面使用面板监控时出现了数据混在一起无法按每台主机进行查看的情况，请问如何才能修改Volume 名称呢

**Reply by zhijian-pro** [reply, marked as answer]
仓库的 main 分支支持了自定义 label，该功能开发完毕但还未发布，其允许覆盖 vol_name 的值 https://github.com/juicedata/juicefs/pull/4312
如果是非重要环境可以编译主线拿到二进制  然后在 mount 的时候加上 `--custom-labels vol_name:xxxxx`
